charts/dvpe-deployment-gloo/templates/certificate.yaml

```diff
@@ -12,6 +12,10 @@ metadata:
   name: {{$serviceName}}-certificate-by-issuer
 spec:
   commonName: {{$firstDomain}}
+  secretTemplate:
+    labels:
+      app: {{$serviceName}}
```

Comment:
What is this `app` label for?
As for my knowledge the `argocd.argoproj.io/instance` label should be enough for ArgoCD.

Reply:
some (not all) Kubernetes resources have also app label...
I think it is not required, so I'll remove it